aibolit/patterns/send_null/send_null.py

```diff
@@ -0,0 +1,27 @@
+import javalang
+from aibolit.utils.ast import AST
+
+
+class SendNull:
+
+    def __init__(self):
+        pass
+
+    def value(self, filename: str):
```

Comment:
I recommend you to add typings to your code. Use typings for method signatures and for declared variables. See more examples in the other patterns.

```
from typing import List
...
def value(self, filename: str) -> List[int]: 
  list: List[int[ = []
...
```

Reply:
Will do